app/scripts/migrations/088.test.ts

```diff
@@ -0,0 +1,638 @@
+import { migrate } from './088';
+
+describe('migration #88', () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: 0 },
```

Comment:
should these versions all be 87? Or does it not really matter?

Reply:
It doesn't matter! But I suppose to go with your previous comment about making these tests more realistic I can change this to 87.